manifests/canary-checker.yaml

```diff
@@ -145,6 +176,49 @@ metadata:
 value: -1
 ---
 apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: canary-checker-ui
+  name: canary-checker-ui
+  namespace: platform-system
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: canary-checker-ui
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: canary-checker-ui
+    spec:
+      containers:
+        - env:
+            - name: ORY_KRATOS_URL
+              value: null
+            - name: BACKEND_URL
+              value: ""
+          image: docker.io/flanksource/canary-checker-ui:v1.0.143
```

Comment:
Don't we need a way to configure this?

Reply:
Should it match the canary-checker version or is it independent?

Reply:
Done